lambda/handlers/intentHandler.js

```
@@ -540,6 +559,129 @@ const DeleteDataIntentHandler = {
+    );
+  },
+  async handle(handlerInput) {
+    const { responseBuilder, attributesManager } = handlerInput;
+    const requestAttributes = attributesManager.getRequestAttributes();
+    const favouriteReciter = Alexa.getSlotValue(
+      handlerInput.requestEnvelope,
+      "favouriteReciter"
+    );
+    console.log("Favourite Reciter: ", favouriteReciter);
+    const reciterIndex = parseInt(favouriteReciter) - 1;
+    if (isNaN(reciterIndex) || reciterIndex < 0 || reciterIndex >= adhaanRecitation.length) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)